src/libraries/System.Private.CoreLib/src/System/Double.cs

```diff
@@ -573,219 +574,140 @@ public static bool IsPow2(double value)
-
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.MaxMagnitude(TSelf, TSelf)" />
-        public static double MaxMagnitude(double x, double y) => Math.MaxMagnitude(x, y);
-
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.MinMagnitude(TSelf, TSelf)" />
-        public static double MinMagnitude(double x, double y) => Math.MinMagnitude(x, y);
-
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.Pow(TSelf, TSelf)" />
-        public static double Pow(double x, double y) => Math.Pow(x, y);
-
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.ReciprocalEstimate(TSelf)" />
-        public static double ReciprocalEstimate(double x) => Math.ReciprocalEstimate(x);
-
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.ReciprocalSqrtEstimate(TSelf)" />
-        public static double ReciprocalSqrtEstimate(double x) => Math.ReciprocalSqrtEstimate(x);
-
         /// <inheritdoc cref="IFloatingPoint{TSelf}.Round(TSelf)" />
         public static double Round(double x) => Math.Round(x);
 
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.Round{TInteger}(TSelf, TInteger)" />
-        public static double Round<TInteger>(double x, TInteger digits)
-            where TInteger : IBinaryInteger<TInteger> => Math.Round(x, int.Create(digits));
+        /// <inheritdoc cref="IFloatingPoint{TSelf}.Round(TSelf, int)" />
+        public static double Round(double x, int digits) => Math.Round(x, digits);
 
         /// <inheritdoc cref="IFloatingPoint{TSelf}.Round(TSelf, MidpointRounding)" />
         public static double Round(double x, MidpointRounding mode) => Math.Round(x, mode);
 
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.Round{TInteger}(TSelf, TInteger, MidpointRounding)" />
-        public static double Round<TInteger>(double x, TInteger digits, MidpointRounding mode)
-            where TInteger : IBinaryInteger<TInteger> => Math.Round(x, int.Create(digits), mode);
+        /// <inheritdoc cref="IFloatingPoint{TSelf}.Round(TSelf, int, MidpointRounding)" />
+        public static double Round(double x, int digits, MidpointRounding mode) => Math.Round(x, digits, mode);
 
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.ScaleB{TInteger}(TSelf, TInteger)" />
-        public static double ScaleB<TInteger>(double x, TInteger n)
-            where TInteger : IBinaryInteger<TInteger> => Math.ScaleB(x, int.Create(n));
+        /// <inheritdoc cref="IFloatingPoint{TSelf}.Truncate(TSelf)" />
+        public static double Truncate(double x) => Math.Truncate(x);
 
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.Sin(TSelf)" />
-        public static double Sin(double x) => Math.Sin(x);
+        //
+        // IFloatingPointIeee754
+        //
 
-        /// <inheritdoc cref="IFloatingPoint{TSelf}.SinCos(TSelf)" />
-        public static (double Sin, double Cos) SinCos(double x) => Math.SinCos(x);
+        /// <inheritdoc cref="IFloatingPointIeee754{TSelf}.E" />
+        static double IFloatingPointIeee754<double>.E => Math.E;
```

Comment:
Style nit: The rest of the IEEE754 properties use the locally re-exposed const.  This seems to be the only one that goes back to `Math.`

```suggestion
        static double IFloatingPointIeee754<double>.E => E;
```